src/components/product-card/product-card.component.html

```diff
@@ -0,0 +1,50 @@
+<section> <!-- I have a section tag here, however, I feel that it may need to be removed if we dynamically add multiple of these components-->
```

Comment:
Agree, I'd just remove it. It's not necessary for the component markup. Sections can just be added wherever we consume this component if needed.